src/test/scala/play/api/cache/redis/test/RedisMasterSlaveContainer.scala

```diff
@@ -0,0 +1,32 @@
+package play.api.cache.redis.test
+
+import com.dimafeng.testcontainers.{Container, DockerComposeContainer, ExposedService}
+import org.scalatest.{BeforeAndAfterAll, Suite}
+import org.testcontainers.containers.wait.strategy.Wait
+
+import java.io.File
+
+trait RedisMasterSlaveContainer extends BeforeAndAfterAll {
```

Comment:
Could you extend `ForAllTestContainer` instead?

Reply:
Hello @KarelCemus ! I inherited `ForAllTestContainer`

Reply:
ou, is it this much incompatible with the interfaces? Sorry for making you to inherit it. I thought it would match. Let's merge it like this and I'll eventually take a look and improve the abstraction.